app/views/demographics/new.html.erb

```diff
@@ -0,0 +1,56 @@
+<h2 class="text-center">Demographics</h2>
+<div class="row">
+  <%= form_tag demographics_path do %>
+    <div class="well">
+      <p class="text-center">To help us make sure we provide all students with equal opportunity, you have the option of disclosing demographic information about yourself.</p>
+      <p class="text-center">Filling this out is voluntary and, if you choose not to, you will not be negatively affected in any way.</p>
+      <hr>
+      <div class="form-group">
+        <%= label_tag :genders, "Gender (check all that apply):" %>
+        <ul class="list-unstyled">
+          <% ["Female", "Male", "Non-binary", "Transgender"].each do |gender| %>
```

Comment:
Yep makes sense! I knew something felt weird about the way I was doing it. It's been awhile since I've worked with non-model-backed models!